src/rules/no_delete_var.rs

```diff
@@ -101,7 +112,13 @@ mod tests {
   fn no_delete_var_invalid() {
     assert_lint_err! {
       NoDeleteVar,
-      r#"var someVar = "someVar"; delete someVar;"#: [{ col: 25, message: MESSAGE, hint: HINT }],
+      r#"var someVar = "someVar"; delete someVar;"#: [
+        {
+          col: 25,
+          message: variant!(NoDeleteVarMessage, Unexpected),
+          hint: variant!(NoDeleteVarHint, Remove),
```

Comment:
This seems superfluous - `NoDeleteVarHint::Remove` is shorter 😃

Reply:
I was wondering which to choose, `variant!(FooEnum, Bar)` or `FooEnum::Bar` if the variant has no value. I don't have strong opinion about this, so let's go with your idea :) I'll rewrite soon.

Reply:
Done